File: benchmarks/programs/primecount/primecount.py
count = 0
for n in range(2, 10001):
    is_prime = True
    d = 2
    while d * d <= n:
        if n % d == 0:
            is_prime = False
            break
        d += 1
    if is_prime:
        count += 1
print(count)

File: benchmarks/programs/primecount/primecount.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    int count = 0;
    for (int n = 2; n <= 10000; n++) {
        bool is_prime = true;
        for (int d = 2; d * d <= n; d++) {
            if (n % d == 0) {
                is_prime = false;
                break;
            }
        }
        if (is_prime)
            count++;
    }
    printf("%d\n", count);
    return 0;
}
